### ISSUE TITLE
pkg_resources reports setuptools present in v1.6 but not v1.7

### ISSUE THREAD

**Issue by aalcock** [pypa/virtualenv#307]
This is reporting a change in behaviour of pkg_resources between v1.6 and v1.7. 

Environment: 
    \* Ubuntu 11.10/Python 2.7.2/setuptools 0.6c11/virtualenv 1.6.4
    \* Ubuntu 12.04LTS/Python 2.7.3/setuptools 0.6c11/virtualenv 1.7.3

In both cases, setuptools 0.6c11 was installed on the system path. A virtualenv was created using the simplest, default arguments::

```
$ virtualenv my_env
```

Then, the following script was run::

```
from pkg_resources import working_set, parse_requirements
for parsed in parse_requirements('setuptools'):
    print parsed, ":", working_set.find(parsed)
```

In the case of v1.6, the code reported 'setuptools : setuptools 0.6c1'. In the case of v1.7 - 'setuptools : None'

In other words, v1.6 is making setuptools available, and v1.7 is hiding the package.

If the v1.7 virtual environment is made with::

```
$ virtualenv --setuptools my_env
```

Then setuptools is detected as expected.
